README.md

```diff
@@ -318,7 +339,7 @@ install this earlier, you can do so with the command:
 sudo apt-get install node-uglify
 ```
 
-## FRDM-K64F Platform
+## FRDM K64F Platform
```

Comment:
I couldn't link the section with the dash. The (#section-name) uses dashes where there are spaces. I am not very proficient at markdown but if you know how to escape the dashes so it will work in the link I will do that.